snapshot.go

```diff
@@ -13,9 +13,19 @@
 
 package firecracker
 
+import "github.com/firecracker-microvm/firecracker-go-sdk/client/models"
+
 type SnapshotConfig struct {
 	MemFilePath         string
+	MemBackend          *models.MemoryBackend
 	SnapshotPath        string
 	EnableDiffSnapshots bool
 	ResumeVM            bool
 }
+
+func (cfg *SnapshotConfig) GetMemFilePath() string {
```

Comment:
wdyt about changing this func name to `GetMemBackendPath`? Since the return here can either point to a file or an uffd.

Reply:
Sounds good, done